packages/material-ui/src/Slider/Slider.js

```diff
@@ -1,252 +1,137 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import clsx from 'clsx';
 import { chainPropTypes } from '@material-ui/utils';
-import withStyles from '../styles/withStyles';
-import useTheme from '../styles/useTheme';
-import { alpha, lighten, darken } from '../styles/colorManipulator';
-import useIsFocusVisible from '../utils/useIsFocusVisible';
-import useEnhancedEffect from '../utils/useEnhancedEffect';
-import ownerDocument from '../utils/ownerDocument';
-import useEventCallback from '../utils/useEventCallback';
-import useForkRef from '../utils/useForkRef';
+import { SliderUnstyled } from '@material-ui/unstyled';
```

Comment:
If I try to change this import to
```suggestion
import SliderUnstyled from '@material-ui/unstyled/SliderUnstyled';
```

the umd build is failing with the following error:

```
./src/index.js → build/umd/material-ui.development.js...
[!] Error: '@material-ui/unstyled/SliderUnstyled' is imported by src/Slider/Slider.js, but could not be resolved – treating it as an external dependency
Error: '@material-ui/unstyled/SliderUnstyled' is imported by src/Slider/Slider.js, but could not be resolved – treating it as an external dependency
    at Object.onwarn (/tmp/425dc60a/packages/material-ui/scripts/rollup.config.js:74:11)
    at Object.config.onwarn.warning [as onwarn] (/tmp/425dc60a/node_modules/rollup/dist/shared/mergeOptions.js:98:25)
    at Object.config.onwarn.warning [as onwarn] (/tmp/425dc60a/node_modules/rollup/dist/shared/rollup.js:19484:20)
    at ModuleLoader.handleResolveId (/tmp/425dc60a/node_modules/rollup/dist/shared/rollup.js:18391:26)
    at Promise.all.Array.from (/tmp/425dc60a/node_modules/rollup/dist/shared/rollup.js:18358:22)
```

Reply:
I could make it work with:

```diff
diff --git a/babel.config.js b/babel.config.js
index b5b77d8501..37b94ca978 100644
--- a/babel.config.js
+++ b/babel.config.js
@@ -114,6 +114,18 @@ module.exports = function getBabelConfig(api) {
           ],
         ],
       },
+      stable: {
+        plugins: [
+          [
+            'babel-plugin-module-resolver',
+            {
+              alias: {
+                '@material-ui/unstyled': '../../packages/material-ui-unstyled/src',
+              },
+            },
+          ],
+        ],
+      },
       legacy: {
         plugins: [
           // IE11 support
diff --git a/packages/material-ui/src/Slider/Slider.js b/packages/material-ui/src/Slider/Slider.js
index 06ec515857..8b9052a809 100644
--- a/packages/material-ui/src/Slider/Slider.js
+++ b/packages/material-ui/src/Slider/Slider.js
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { chainPropTypes } from '@material-ui/utils';
-import { SliderUnstyled } from '@material-ui/unstyled';
+import SliderUnstyled from '@material-ui/unstyled/SliderUnstyled';
 import { useThemeProps, experimentalStyled, alpha, lighten, darken } from '../styles';
 import capitalize from '../utils/capitalize';
 import ValueLabel from './ValueLabel';
```

Reply:
Let me try it, for some reason I have missed this comment